### PR TITLE
2.0 release

### LINKED ISSUE #18: move assets version to higher level
for now, modules offersList and offersAlert make use of assets and inject them with Combiner which requires a version number for cache management. 
image

$strVersion should be defined once, and at a higher level to avoid having to go fishing each update

### PR DESCRIPTION
**Pull request by LupusVII**
Bump dependencies and adapt bundle code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Extension "Job Offers" for Contao Open Source CMS
+Extension "Offers" for Contao Open Source CMS
 ========
+X.Y.Z - YYYY-MM-DD
```
Comment (ChipsVII) [inline CHANGELOG.md]: Fix the version & date

File: src/Hooks/LoadDataContainerHook.php
Comment (ChipsVII) [inline src/Hooks/LoadDataContainerHook.php:1]: As we are in a code refactoring, shouldn't we move from `Hooks` to `EventListener` folder ?

File: src/Resources/contao/config/config.php [hunk 2]
```diff
@@ -33,66 +34,74 @@
...
-    } else {
-        $GLOBALS['TL_CSS'][] = 'bundles/joboffers/backend/backend_svg.css';
-    }
-}
-
 // Hooks
-$GLOBALS['TL_HOOKS']['loadDataContainer'][] = [WEM\JobOffersBundle\Hooks\LoadDataContainerHook::class, 'addAttributesToJobDca'];
-$GLOBALS['TL_HOOKS']['storeFormData'][] = [WEM\JobOffersBundle\Hooks\StoreFormDataHook::class, 'storeFormData'];
-$GLOBALS['TL_HOOKS']['processFormData'][] = [WEM\JobOffersBundle\Hooks\ProcessFormDataHook::class, '__invoke'];
+$GLOBALS['TL_HOOKS']['loadDataContainer'][] = [WEM\OffersBundle\Hooks\LoadDataContainerHook::class, 'addAttributesToJobDca'];
```
Comment (ChipsVII) [inline src/Resources/contao/config/config.php]: As we're  in a code refactoring, shouldn't we use the new syntax `offers.listener.xxxx` ?

**Review by ChipsVII — CHANGES_REQUESTED** [review]
Is it testable somewhere ?

File: CHANGELOG.md (hunk 1 above)
Comment (LupusVII) [reply CHANGELOG.md]: Fixed in 9dd41f1febc20264f9a9ad0c4e4df8afc3118e04

File: src/Hooks/LoadDataContainerHook.php
Comment (LupusVII) [reply src/Hooks/LoadDataContainerHook.php:1]: Fixed in 67de83cbdbf7a6bc09ec197355649082da765d9d

File: src/Resources/contao/config/config.php (hunk 2 above)
Comment (LupusVII) [reply src/Resources/contao/config/config.php]: Fixed in 67de83cbdbf7a6bc09ec197355649082da765d9d

**Review by ChipsVII — APPROVED** [review]

File: src/Resources/contao/templates/offer_default.html5 [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+							</div>
+						<?php endif ?>
+					<?php endforeach ?>
+				</div>
+				<?php endif ?>
+			</div>
+			<div class="offer__actions">
+				<<?php if ($this->blnDisplayApplyButton): ?>button<?php else: ?>div<?php endif; ?> class="btn offer__action <?php if (!$this->blnDisplayApplyButton): ?>ft-primary<?php endif; ?>" title="<?= $GLOBALS['TL_LANG']['WEM']['OFFERS']['seeOffer'] ?>" data-process="openDetailsModal" data-offer="<?= $this->id ?>" data-icon="false" data-result="false">
+					<?php if ($this->blnDisplayApplyButton): ?>
+					<?= $GLOBALS['TL_LANG']['WEM']['OFFERS']['seeOffer'] ?>
```
Comment (ChipsVII) [inline src/Resources/contao/templates/offer_default.html5:35]: So if `$this->blnDisplayApplyButton` is `false`, we display a `div` with no text ?

**Pull request merged**